examples/react/src/components/Content.tsx

```diff
@@ -24,6 +25,18 @@ const Content: React.FC = () => {
   const [messages, setMessages] = useState<Array<Message>>([]);
   const [loading, setLoading] = useState<boolean>(false);
 
+  // Try eager connect on app launch
+  useEffect(() => {
+    const eagerConnect = async () => {
```

Comment:
Why is this code needed?

If you already are `signedIn` with a wallet and the `selectedWalletId` has a value we check that internally when we setup wallet modules.

Reply:
Nightly and I expect other wallet need some kind of session. You need to call `signIn()` or wallet will not able to receive informations from webpage. This is how injected wallets should work IMO.
Nightly knows if you already got permission to connect so popup will not show again.

Reply:
Yes if wallet is locked this will trigger popup to unlock. Why is that an issue ? This is common behaviour.

Reply:
This flow doesn't occur with the other two injected wallets (Sender and Math Wallet). It's an issue because a user will navigate to a dApp they previously signed in to using Wallet Selector and immediately see the unlock prompt

Reply:
I am not focusing API on near alone. I am trying to fit into all blockchains (Solana/Near now) and on Solana this is common.
If you guys insist I will create a method that connects only if user already approved app and wallet is unlocked.